velox/docs/functions/spark/aggregate.rst

```diff
@@ -80,6 +80,14 @@ General Aggregate Functions
 
     Returns the last non-null value of `x`.
 
+.. spark:function:: max(x) -> [same as x]
+
+    Returns the maximum value of ``x``.
+    ``x`` must be an orderable type.
+    Note: 1. the timestamp type represents a time instant in microsecond precision.
```

Comment:
> the timestamp type represents a time instant in microsecond precision

nit: perhaps make the sentence more concise and capitalize the first letter: Timestamp is in microsecond precision.